src/utils/ao.ts

```diff
@@ -59,6 +60,12 @@ export async function spawnANT({
   stateContractTxId?: string;
   antRegistryId?: string;
 }): Promise<string> {
+  // AoSigner is not a Contract Signer - should probably add that to the contract signer type
+  const registryClient = new AOProcess({
+    processId: antRegistryId,
+    ao,
+  });
```

Comment:
@atticusofsparta - can you come back and update this now?

Reply:
https://github.com/ar-io/ar-io-sdk/pull/191